cirq/protocols/resolve_parameters.py

```diff
@@ -73,6 +75,15 @@ def resolve_parameters(val: Any, param_resolver: 'cirq.ParamResolver') -> Any:
         If `val` has no `_resolve_parameters_` method or if it returns
         NotImplemented, `val` itself is returned.
     """
+    # HACK: break cycle.
+    from cirq import ParamResolver
```

Comment:
nit: move down to right before it is used?  Not sure...imports in code are strange.

Reply:
Done.